src/__internal__/focus-trap/focus-trap.spec.js

```diff
@@ -428,4 +429,59 @@ describe("FocusTrap", () => {
       }).not.toThrow();
     });
   });
+
+  describe("when content in the children tree changes", () => {
+    it("should trigger the MutationObserver", () => {
+      const mutationObserverMock = jest.fn(function MutationObserver(callback) {
+        this.observe = jest.fn();
+        this.disconnect = jest.fn();
+        this.trigger = () => {
+          callback();
+        };
+      });
+      global.MutationObserver = mutationObserverMock;
```

Comment:
Do we not need to reset `global.MutationObserver` after the test has run?

Reply:
We do indeed